engine/security/trust/trust_key_mng.md

```diff
@@ -69,16 +71,18 @@ for more details.
 
 Prior to Docker Engine 1.11, this feature was only in the experimental branch.
 
-## Lost keys
+## Key Loss
 
-If a publisher loses keys it means losing the ability to sign trusted content for
-your repositories.  If you lose a key, send an email to [Docker Hub
-Support](mailto:hub-support@docker.com) to reset the repository
-state.
+> **WARNING**
+>
+> If a publisher loses keys it means losing the ability to sign trusted content for the repositories in
```

Comment:
I would probably steer clear of the term "trusted content" when we are talking about signed software. Use trusted content for DOI/DVP/DSOS.

```suggestion
> If a publisher loses keys it means losing the ability to sign images for the repositories in
```

Reply:
Addressed in commit 3.